src/measure_theory/integral/circle_transform.lean

```diff
@@ -174,4 +173,308 @@ begin
   exact this,
 end
 
+/--Cauchy integral form of a function at `z` in a disk of radius `R`-/
+def circle_integral_form [complete_space E] (R : ℝ) (z : ℂ) (f : ℂ → E) : (ℂ → E) :=
+  λ w, (2 * π * I : ℂ)⁻¹ • (∮ z in C(z, R), (z - w)⁻¹ • f z)
+
+lemma circle_intgral_form_eq_int [complete_space E] (R : ℝ) (z : ℂ) (f : ℂ → E) :
+  circle_integral_form R z f = λ w,
+ ∫ (θ : ℝ) in 0..2 * π, (circle_transform R z w f) θ :=
+begin
+    simp_rw [circle_transform, circle_integral_form, circle_integral,
+      interval_integral.integral_smul],
+end
+
+lemma circle_transform_circle_int [complete_space E] (R : ℝ) (z w : ℂ) (f : ℂ → E) :
+  ∫ (θ : ℝ) in 0..2 * π, circle_transform R z w f θ =
+ (2 * π * I : ℂ)⁻¹ • ∮ z in C(z, R), (z - w)⁻¹ • f z :=
+begin
+  simp_rw [circle_transform, circle_integral,deriv_circle_map, circle_map],
+  simp only [real_smul, nsmul_eq_mul, nat.cast_bit0, nat.cast_one, one_div,
+  interval_integral.integral_smul, zero_add],
```

Comment:
```suggestion
    interval_integral.integral_smul, zero_add],
```
I would guess that you can put the lemmas from `simp_rw` into the `simp only` and then write everything as `by simp only [..]`